src/environment/stella_environment.hpp

```diff
@@ -30,6 +30,8 @@
 
 #include <stack>
 
+class StellaEnvironment;
```

Comment:
This forward declaration is spurious, no?

Reply:
You are right. Fixed.